### DISCUSSION TITLE
Kits

### DISCUSSION THREAD

**Discussion by Karonas** [EssentialsX/Essentials#4622] [Q&A] [answered]
Hello, so all of my kits are in order on kits.yml but on the server,
 they are not in the order. Is there anything I can do so that they could be in order like on the kits.yml file
![image_2021-11-09_123316](https://user-images.githubusercontent.com/93981152/140908479-eba69089-66f4-4e8f-8992-fa15d8cf517c.png)
![image_2021-11-09_123331](https://user-images.githubusercontent.com/93981152/140908514-a3fa8fba-9e8e-4f83-9969-b3f98acf6708.png)

**Comment by JRoy** [comment, marked as answer]
There will be no guaranteed ordering for kits as there are memory implications for this.
